### PR TITLE
Scavengers: Incognito + filter improvements

### PR DESCRIPTION
**Pull request by sparkychildcharlie**

### CODE REVIEW

**Comment by sparkychildcharlie** [comment]
(as usual - too lazy to work with branches and push)

File: chat-plugins/scavengers.js [hunk 1]
```diff
@@ -1373,6 +1373,46 @@ let commands = {
 		this.privateModAction(`(${user.name} has ${(change > 0 ? 'given' : 'taken')} one infraction point ${(change > 0 ? 'to' : 'from')} '${targetId}'.)`);
 		this.modlog(`SCAV ${this.cmd.toUpperCase()}`, user);
 	},
+
+	modsettings: {
+		'': 'update',
+		'update': function (target, room, user) {
+			if (!this.can('declare') || room.id !== 'scavengers') return false;
```
Comment (whalemer) [inline chat-plugins/scavengers.js]: does this need to check global perms?

Comment (sparkychildcharlie) [reply chat-plugins/scavengers.js]: oh oops havent touched ps code in a looong time - ill fix when i get on a computer

**Comment by sparkychildcharlie** [comment]
Fixed xD - the cons of testing using a sysop/admin account

**Pull request merged**